docs/usingwidgets/index.html

```diff
@@ -396,7 +396,7 @@ <h3 id="adding-widgets">Adding widgets</h3>
 app.display()
 </code></pre>
 <p>Let’s look at the <a href="../text/">Text</a> widget code in a bit more detail:</p>
-<pre><code class="language-python">message = Text(app, text=&quot;Enter your name&quot;))
+<pre><code class="language-python">message = Text(app, text=&quot;Enter your name&quot;)
```

Comment:
You didnt need to modify the HTML, it will get regenerated when a release is done. But it doesnt hurt so no problem.

Reply:
So i guess, you gonna merge it actually ?